src/backend/abuseDetection.ts

```diff
@@ -66,8 +66,8 @@ export function createDynamoDbClient(
     // Increments the integer value at given key.
     // If the key doesn't exist, it's created automatically as having value 0, then incremented normally.
     // Returns the integer value (after being incremented).
-    incrementKey(key: string): Promise<number> {
-      return ddb
+    async incrementKey(key: string): Promise<number> {
```

Comment:
I was bit surprised myself that this syntax actually works on object literals 😏

Reply:
Oh, yeah, same here :sweat_smile: